doc/man/8/rbd.rst

```diff
@@ -10,7 +10,7 @@ Synopsis
 ========
 
 | **rbd** [ -c *ceph.conf* ] [ -m *monaddr* ] [ -p | --pool *pool* ] [
-  --size *size* ] [ --order *bits* ] [ *command* ... ]
+  --size *size* ] [ --order *bits* ] [ *command* ... ] [--cluster *cluster name*]
```

Comment:
Minor: can you move ```--cluster xyz``` to near the front, by ```-c``` and ```-m``` options, since that would group the bootstrap options?